### PR TITLE
Fix/signup troubleshooting

### PR DESCRIPTION
**Pull request by reyesrene**

### CODE REVIEW

File: models/_model.js [hunk 1]
```diff
@@ -418,24 +428,30 @@ class Model {
           gravityCLIReporter.addItemsInJson('New Record Account', self.record , `NEW ${self.model} RECORD`);
 
           let encryptedRecord;
+          console.log(1)
```
Comment (joramirezStackit) [inline models/_model.js:431]: Avoid using console logs and debugging info

File: workers/registration.js [hunk 2]
```diff
@@ -1,34 +1,47 @@
...
     const database = response.database || response.tables;
     // const { tableList } = response;
     const tableBreakdown = gravity.tableBreakdown(database);
 
     if (response.error) {
+
+      logger.debug(`checkRegistration().getUser() > response error: ${response.error}`);
       done();
+      //@TODO the following will trigger this function to run again. this means we can get into an infinite loop.
+      console.log(1)
```
Comment (joramirezStackit) [inline workers/registration.js:43]: avoid using console.log and debigging info

File: workers/registration.js [hunk 3]
```diff
@@ -58,15 +71,19 @@ class RegistrationWorker extends Worker {
...
       if (res.error) {
-        console.log(res.error);
+        logger.error(res.error);
         if (res.fullError === 'Error: Unable to save table. users is already in the database') {
           data.usersExists = true;
           data.usersConfirmed = false;
         }
       }
       done();
+      console.log(2)
```
Comment (joramirezStackit) [inline workers/registration.js:81]: same here

File: workers/registration.js [hunk 4]
```diff
@@ -134,8 +161,12 @@ class RegistrationWorker extends Worker {
         }
       }
       done();
+      console.log(6)
```
Comment (joramirezStackit) [inline workers/registration.js:164]: all console.log and debugging info should be removed

**Review by joramirezStackit — CHANGES_REQUESTED** [review]

File: models/_model.js (hunk 1 above)
Comment (reyesrene) [reply models/_model.js:431]: ok

**Review by joramirezStackit — APPROVED** [review]
LGTM

**Pull request merged**